OpenRA.Mods.Common/Widgets/Logic/Installation/DownloadPackagesLogic.cs

```diff
@@ -52,9 +57,9 @@ void ShowDownloadDialog()
 
 			var cancelButton = panel.Get<ButtonWidget>("CANCEL_BUTTON");
 
-			var mirrorsFile = Platform.ResolvePath("^", "Content", Game.ModData.Manifest.Mod.Id, "mirrors.txt");
+			var mirrorsFile = Platform.ResolvePath("^", "Content", "mirrors.txt");
```

Comment:
Or even better, we could go with my suggestion (which was ignored) from the original PR, and use the version of the `Download` constructor that returns a string instead of saving a file to disk.